arclines/holy/patterns.py

```diff
@@ -222,3 +223,182 @@ def score_quad_matches(fidx):
             scores.append('Amb')
     # Return
     return scores
+
+
+def triangles(detlines, linelist, detsrch, lstsrch, npixels, pixtol):
+    """
+    Parameters
+    ----------
+    detlines : ndarray
+      list of detected lines in pixels (sorted, increasing)
+    linelist : ndarray
+      list of lines that should be detected (sorted, increasing)
+    detsrch : int
```

Comment:
Maybe initiaize this to a preferred value?

Reply:
Good idea - done.